Fitness/Controllers/OnboardingViewController.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  PresntationExampleViewController.swift
+//  Fitness
+//
+//  Created by Yassin Mziya on 10/6/18.
+//  Copyright © 2018 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+import Presentation
+import SnapKit
+
+class OnboardingViewController: PresentationController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+        view.backgroundColor = .white
+        
+        configureSlides()
+        configureBackground()
+    }
+
+    override init(transitionStyle style: UIPageViewControllerTransitionStyle, navigationOrientation: UIPageViewControllerNavigationOrientation, options: [String : Any]? = nil) {
+        super.init(transitionStyle: .scroll, navigationOrientation: .horizontal)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    @objc func dismissOnboarding() {
+        print("superbad, call me mclovin")
```

Comment:
Probably should remove prints [lol](https://img.maximummedia.ie/her_ie/eyJkYXRhIjoie1widXJsXCI6XCJodHRwOlxcXC9cXFwvbWVkaWEtaGVyLm1heGltdW1tZWRpYS5pZS5zMy5hbWF6b25hd3MuY29tXFxcL3dwLWNvbnRlbnRcXFwvdXBsb2Fkc1xcXC8yMDE3XFxcLzA4XFxcLzE4MTUxMTE1XFxcL2hxZGVmYXVsdC5qcGdcIixcIndpZHRoXCI6NjQwLFwiaGVpZ2h0XCI6MzYwLFwiZGVmYXVsdFwiOlwiaHR0cHM6XFxcL1xcXC93d3cuaGVyLmllXFxcL2Fzc2V0c1xcXC9pbWFnZXNcXFwvaGVyXFxcL25vLWltYWdlLnBuZz92PTVcIn0iLCJoYXNoIjoiNjZmYzEyNTUzOWU4NzgxZjJkZTA0NThjOGVmNjNlODBlOTVmNDMwYiJ9/hqdefault.jpg)